packaging/convert2rhel.spec

```diff
@@ -122,6 +122,56 @@ install -m 0600 config/convert2rhel.ini %{buildroot}%{_sysconfdir}/convert2rhel.
 %attr(0644,root,root) %{_mandir}/man8/%{name}.8*
 
 %changelog
+* Mon May 27 2024 Adam Hosek <ahosek@redhat.com> 2.0.0
+- Remove deprecated CLI arguments
+- Remove deprecated latest kernel check env variable
+- Remove deprecated convert2rhel unsupported version env variable
+- Remove `--disable-submgr` arg
+- Rename incomplete rollback envvar
+- Remove `-v|--variant` arg
+- Port RestorablePackage and ChangedRPMPackagesController
+- Remove backup controller partition
+- Update RHSM custom facts during analysis
+- Remove ppc64 architecture from data
+- Add debug logs to host-metering
+- Add post conversion stages for Action Framework
+- Set consistent task prefix for pre-conversion
+- Migrate post conversion transaction function
+- Improve efi error check for PART_ENTRY_NUMBER
+- Unify parsing of the system release string
+- Drop support for 8.6 EUS conversions
+- Improve detection of RHSM credentials in subscription check
+- Update report to include post conversion actions
+- Satellite package backup failure fix
+- Drop internet connection check and its dependencies
+- Don't report on skipping dbus check
+- Modify exit code for conversion when inhibitor is found
+- Add RHEL 9 version support up to 9.10
+- Allow converting to RHEL 7 ELS
+- Exit with 1 when rollback fails
+- Fix empty error message in Yum transaction failure
+- Disable RHEL repos when performing checks
+- Update the Red Hat package signing key
+- Install subscription-manager using package manager
+- Change result of the c2r version check to OVERRIDABLE
+- Restore disabled repos during conversion in rollback
+- Rollback task using "Convert" instead of "Rollback"
+- Override yum config exclude list
+- Fix an error message referring to a non-existing --organization option
+- Fix RHSM facts filepath in a log message
+- Remove leftover reposdir from package updates action
+- Require username and password to be specified
+- Fix check for reposdir in RestorablePackage class
+- Add the libreport-plugin-mantisbt to excluded list
+- Always backup redhat.repo
+- Fix cornercase with userpass and keyorg
+- Fix report duplication after analysis
+- Fix regression in installing removed packages during rollback
+- Check if rollback failed when handling InhibitorFound exception
+- Skip versionlock file in backup system actions
+- Raise when RestorablePackage fails to restore
+- Return a list instead of string for disablerepo command
+- Fix bug with duplicate packages check when running with Satellite offline
```

Comment:
I'd reword all the breaking changes and have them near the top

Reply:
@Venefilyn Like having `Breaking change:` in the beginning? And I think all of the are on the top, I haven't changed the order

Reply:
Yep!